File: App/src/app/status/entities/status.entity.ts
import { TaskEntity } from '../../task/entities/task.entity';
import { BaseEntity } from '../../shared/entities/base.entity';
import { ProjectStatusEntity } from '../../shared/entities/projectStatus.entity';

export interface StatusEntity extends BaseEntity {
  name: string;
  color: string;
  task?: TaskEntity;
  projectStatus?: ProjectStatusEntity[];
}

File: Api/src/Api.Domain/Entities/StatusEntity.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Domain.Entities {
    public class StatusEntity : BaseEntity 
    {
        [Required]
        [MaxLength(100)]
        public string Name { get; set; }

        [Required]
        [MaxLength(7)]
        public string Color { get; set; }

        [JsonIgnore]
        public IEnumerable<TaskEntity> Task { get; set; }

        [JsonIgnore]
        public IEnumerable<ProjectStatusEntity> ProjectStatus { get; set; }
    }
}
